### PR TITLE
Adds distance check to removing lights from their fixtures as a borg

### LINKED ISSUE #4300: borgs can shift+click lights to remove the light tube, even from a distance
Submitter: matlap
**Describe the bug**

**To Reproduce**
1. be a borg
2. shift click a light fixture
3. the light is removed and appears under your feet

**Expected behavior**
nothing happens

Reported on: Goonstation 1 Classic: Heisenbee 2021-04-11 22:30:13

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Adds distance check to removing lights from their fixtures as a borg, fixes #4300

### CODE REVIEW

File: code/modules/power/lighting.dm [hunk 1]
```diff
@@ -783,6 +783,8 @@
 		else
 			prot = 1
 
+		if (get_dist(src, user) > 1)
```
Comment (ZeWaka) [inline code/modules/power/lighting.dm:786]: please use the macro

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/modules/power/lighting.dm (hunk 1 above)
Comment (Carbadox) [reply code/modules/power/lighting.dm:786]: if you mean in_interact_range then I did. I tested it and it didn't work whatsoever. It still let me take out lights

Comment (ZeWaka) [reply code/modules/power/lighting.dm:786]: i actually meant `IN_RANGE`, but `in_interact_range` is indeed the correct one to use in this case.

Comment (Carbadox) [reply code/modules/power/lighting.dm:786]: Same issue

Comment (ZeWaka) [reply code/modules/power/lighting.dm:786]: unable to reproduce. I literally changed the line locally and it worked fine. you probably forgot to negate it?

Comment (ZeWaka) [inline code/modules/power/lighting.dm:786]:
```suggestion
		if (!in_interact_range(src, user))
```

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Pull request merged**